### PR TITLE
Feature/add license

### PR DESCRIPTION
**Pull request by joaquinmarti**

### CODE REVIEW

**Review by muryoh — APPROVED** [review]

File: package.json [hunk 1]
```diff
@@ -1,4 +1,5 @@
 {
- "name": "@components-studio/divriots.studio-story-renderer",
- "version": "0.0.0"
+  "name": "@components-studio/divriots.studio-story-renderer",
```
Comment (gqio) [inline package.json:2]: I think it should be @divriots/studio-story-rendere

Comment (gqio) [reply package.json:2]: @muryoh  :D

Comment (muryoh) [reply package.json:2]: Heh we'll fix it :+1: 
It's ok in the released package though

image

Comment (gqio) [reply package.json:2]: ok 👍

**Pull request merged**